infoblox/resource_infoblox_ip_allocation.go

```diff
@@ -110,79 +109,60 @@ func resourceIPAllocation() *schema.Resource {
 					" used by Infoblox Terraform plugin to search for a NIOS's object" +
 					" which corresponds to the Terraform resource.",
 			},
+			"ref": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "NIOS object's reference, not to be set by a user.",
+			},
 		},
 	}
 }
 
-func getAndRenewHostRecAltId(d *schema.ResourceData, m interface{}, updateId bool) (hostRec *ibclient.HostRecord, err error) {
+// This function is for retrieving a host record by either known reference or,
```

Comment:
According to [godoc](https://go.dev/blog/godoc), comment should begin with the name of the element it describes. 

```suggestion
// getOrFindHostRec is for retrieving a host record by either known reference or,
```